interfaces/IBF-dashboard/src/app/services/event.service.ts

```diff
@@ -178,6 +180,19 @@ export class EventService {
     event.firstLeadTimeLabel = LeadTimeTriggerKey[firstKey];
     event.timeUnit = firstKey?.split('-')[1];
 
+    // event.startDate = '2022-02-01';
+    // event.startDateEvent = '2022-02-07';
+    // event.lastModelRunDate = '2022-02-09';
```

Comment:
@arsforza I added these commented lines, to be able to test different scenarios. Play with these values to properly review this item. I made a separate subtask to remove these commented lines again after reviewing.